src/parser/um_parser.py

```
@@ -28,6 +30,147 @@ class UMParser:
     # 1 箇所で検算するために使う（ここが CR/LF なら手前の全項目が正しく閉じている）
     RECORD_DELIMITER_START = 1607
 
+    # 公式コード表の桁数。項番14/15/16/17/19（馬記号・性別・品種・毛色・東西所属）は
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)